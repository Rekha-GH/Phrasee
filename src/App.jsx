import React from 'react';
import 'antd/dist/antd.css';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from "react-router-dom";
import LoginPage from './Login/Components/LoginPage';
import ProvideAuth from './Common/Components/ProvideAuth';
import { Provider } from 'react-redux';
import store from './Store/Store';
import PrivateRoute from './Common/Components/PrivateRoute';
import PatientPage from './Patient/Components/PatientPage';

const App = () => {
  return (
    <Provider store={store}>
      <header>
        <img src="./images/dhg_whole.png" alt={'logo'} />
      </header>
      <ProvideAuth>
        <Router>
          <Switch>
            <Route path="/login">
              <LoginPage />
            </Route>
            <PrivateRoute path="/">
              <PatientPage />
            </PrivateRoute>
          </Switch>
        </Router>
      </ProvideAuth>
      <footer>
      </footer>
    </Provider>
  );
}

export default App;
