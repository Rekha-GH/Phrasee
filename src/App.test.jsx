import React from 'react';
import toJson from 'enzyme-to-json';

import App from './App';
import { shallow } from './setupTests';

describe('App Tests', () => {
  test('App renders correctly', () => {
    const wrapper = shallow(<App />);
    
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})
