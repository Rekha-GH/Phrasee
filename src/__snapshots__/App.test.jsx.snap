// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Tests App renders correctly 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <header>
    <img
      alt="logo"
      src="./images/dhg_whole.png"
    />
  </header>
  <ProvideAuth>
    <BrowserRouter>
      <Switch>
        <Route
          path="/login"
        >
          <LoginPage />
        </Route>
        <PrivateRoute
          path="/"
        >
          <PatientPage />
        </PrivateRoute>
      </Switch>
    </BrowserRouter>
  </ProvideAuth>
  <footer />
</Provider>
`;
